cfme/tests/intelligence/test_chargeback.py

```diff
@@ -19,20 +19,27 @@
 
 def new_compute_rate():
     return cb.ComputeRate(description='cb' + fauxfactory.gen_alphanumeric(),
-                          cpu_alloc=(1000, cb.HOURLY),
-                          disk_io=(10, cb.HOURLY),
-                          compute_fixed_1=(100, cb.HOURLY),
-                          compute_fixed_2=(200, cb.HOURLY),
-                          mem_alloc=(10000, cb.HOURLY),
-                          mem_used=(4000, cb.HOURLY),
-                          net_io=(6000, cb.HOURLY))
+                          fields=[{'name': 'Allocated CPU Count',
```

Comment:
My first try with parametrized views and locators, so I could miss something. Thank you